recipes/pyode/build.sh

```diff
@@ -0,0 +1,14 @@
+# part 1, library, cmake and c++
+mkdir _build
+cd _build
+
+cmake -DODE_WITH_DEMOS:BOOL=OFF \
+      -DODE_WITH_TESTS:BOOL=OFF .. \
+      -DCMAKE_INSTALL_PREFIX:PATH=""
```

Comment:
```suggestion
      -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX \
      -DCMAKE_PREFIX_PATH=$PREFIX
```

Reply:
This failed when I tried it (& removed the DESTDIR). Looks purer my way; install under basically /lib and /include, and in the PREFIX location